index.html

```diff
@@ -122,9 +132,9 @@ <h3>Join our Slack community!</h3>
             }
           };
 
-          wontAbuse.addEventListener("change", validate);
-          wontSpam.addEventListener("change", validate);
-          willUphold.addEventListener("change", validate);
+          for (const input of formInputs) {
+            input.addEventListener("change", validate);
+          }
```

Comment:
I'd put a single listener on the `#signup` element.

```suggestion
            document.getElementById('signup').addEventListener('change', validate);
```

This makes the `formInputs` variable above redundant.